eessi-2023.06-known-issues.yml

```diff
@@ -17,6 +17,9 @@
   - OpenBLAS-0.3.21-GCC-12.2.0:
     - issue: https://github.com/EESSI/software-layer/issues/314
     - info: "Increased number of numerical errors in OpenBLAS test suite (344 vs max. 150 on x86_64/*)"
+  - PyTorch-2.1.2-foss-2023a:
+    - issue: https://github.com/EESSI/software-layer/issues/461
+    - info: "8 failing tests (out of 209539) on aarch64/*"
```

Comment:
@bedroge can you add a similiar entry for `aarch64/generic` and `aarch64/neoverse_n1`, since this block is specific to `aarch64/neoverse_v1`?

Reply:
Oops, completely overlooked that. I've duplicated the issue for the other aarch64 archs.